tests/cli/test_commands.py

```diff
@@ -1,46 +0,0 @@
-# -*- coding: utf-8 -*-
-"""Tests for CLI commands."""
-from __future__ import annotations
-
-import subprocess
-
-from aiida_pseudo.cli import cmd_root
```

Comment:
This was indeed a mistake, testing the `aiida_pseudo` cli. But instead of removing this entirely, I would simply keep it and actually test the CLI of this package, as intended:
```
from aiida_quantumespresso.cli import cmd_root
```

Reply:
Done, (I didn't realize you have CLI even here :-D)